hardware/kicad/.gitignore

```diff
@@ -0,0 +1,4 @@
+lib_ext
+fromGerber.*
+*-backups
+3D/**
```

Comment:
Could we use [this .gitignore](https://github.com/github/gitignore/blob/main/KiCad.gitignore) instead to avoid temprary/cache files?